### ISSUE TITLE
My IP address is being blocked

### ISSUE THREAD

**Issue by Thangthebrandlab** [allegro/allegro-api#10342]
Hi,

I have been running a stock querying system using the endpoint GET /fulfillment/stock for the past two weeks. However, I am now encountering an issue where I can no longer query data from the Allegro server. Even when testing the query in Postman, I am consistently redirected to a "You have been blocked" geo captcha page.

I am unsure what might be causing this issue. My refresh token appears to work fine, and I am successfully receiving both an access token and a 200 OK response when requesting a new access token via the refresh token endpoint.

Could you please look into this? For reference, this is the ID displayed on the "You have been blocked" page:
ID: c526f95c-b9ac-5145-f707-43220e6350e0

Thank you for your assistance,
Thang Le.

**Comment by Lukasz-Zurek** [comment]
Please provide IP address. If you don't want to provide it here, you can do so via the contact form. In the ticket, add the GitHub thread number - #10342

**Comment by Thangthebrandlab** [comment]
Thank you, I have created a new case with Contact form ID 46649638. I've also added the Github Thread number there.

**Comment by Lukasz-Zurek** [comment]
@Thangthebrandlab I have just received your message and forwarded this case to another department for a verification. As soon as I get some details, I will let you know.

**Comment by Thangthebrandlab** [comment]
Good morning, has there perhaps been an update regarding this matter? Thank you!

**Comment by Lukasz-Zurek** [comment]
@Thangthebrandlab Yes. You've been sending your requests to /fulfillment/stock but to allegro.pl not an API. You should use our API.

**Comment by Thangthebrandlab** [comment]
@Lukasz-Zurek Thank you for getting back. that means our IP address is not blocked. Can you perhaps clarify what you meant API again?

The endpoint I used was https://allegro.pl/fulfillment/stock, this is incorrect?

**Comment by Lukasz-Zurek** [comment]
@Thangthebrandlab For our endpoints available in Allegro API, you should call https://api.allegro.pl/, not https://allegro.pl/.

**Comment by Thangthebrandlab** [comment]
Thank you! This issue is resolved.
